docs/modules/ROOT/pages/contributing/developers.adoc

```diff
@@ -85,29 +85,14 @@ make STAGING_IMAGE_NAME='docker.io/myrepo/camel-k' images-push-staging
 == Testing
 
 Unit tests are executed automatically as part of the build. They use the standard go testing framework.
+The **convention** used in this repo is to name tests `xxx_test.go`.
```

Comment:
I think it's a convention from Golang, that relies on it for `go test`, rather than a convention from this repository. I'd suggest to remove it.

Reply:
Cleaned up!

Reply:
Thanks!